lib/player/player.go

```diff
@@ -135,18 +140,19 @@ func New(cfg *Config) (*Player, error) {
 	)
 
 	p := &Player{
-		clock:     clk,
-		log:       log,
-		sessionID: cfg.SessionID,
-		streamer:  cfg.Streamer,
-		emit:      make(chan events.AuditEvent, 1024),
-		playPause: make(chan chan struct{}, 1),
-		wake:      make(chan int64),
-		done:      make(chan struct{}),
+		clock:        clk,
+		log:          log,
+		sessionID:    cfg.SessionID,
+		streamer:     cfg.Streamer,
+		emit:         make(chan events.AuditEvent, 1024),
+		playPause:    make(chan chan struct{}, 1),
+		wake:         make(chan time.Duration),
+		done:         make(chan struct{}),
+		skipIdleTime: cfg.SkipIdleTime,
 	}
 
 	p.speed.Store(float64(defaultPlaybackSpeed))
-	p.advanceTo.Store(normalPlayback)
+	p.advanceTo.Store(normalPlayback.Milliseconds())
```

Comment:
There's no need to round to milliseconds every time; `time.Duration` is already an int64 storing nanoseconds, you can just do this instead:

```suggestion
	p.advanceTo.Store(int64(normalPlayback))
```